spring-pulsar-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -535,6 +542,43 @@ public Map<String, Object> buildProperties() {
 
 			return properties;
 		}
+
+	}
+
+	public static class Cache {
+
+		/** Time period to expire unused entries in the cache. */
+		private Duration expireAfterAccess = Duration.ofMinutes(1);
```

Comment:
I left out any validation (`@NonNegative` etc...) as Caffeine is the final arbiter and does a very solid job of validating the inputs.